autonomous/rotate.py

```diff
@@ -11,6 +11,6 @@ class Rotate(AutonomousStateMachine):
 
     @state(first=True)
     def rotate(self):
-        if self.drivetrain.rotate(degrees=-90):
+        if self.drivetrain.rotate(degrees=-90, max_speed=90):
```

Comment:
Why 90?

Reply:
You had it set to something like 1.5 radians/second which is roughly 90 degrees/second, more like 85 I believe. I can change it to 85 so the tuning is right where you left it if you want.

Reply:
Oh, ok.